tskv/src/tseries_family.rs

```diff
@@ -1209,6 +1217,25 @@ impl TseriesFamily {
             .sum::<u64>()
             + self.mut_cache.read().cache_size()
     }
+
+    async fn update_vnode_metrics(tsfamily: Arc<TokioRwLock<TseriesFamily>>) {
+        let start = tokio::time::Instant::now() + Duration::from_secs(10);
+        let interval = Duration::from_secs(10);
+        let mut intv = tokio::time::interval_at(start, interval);
+        loop {
+            intv.tick().await;
+            let cache_size = {
+                let tsfamily = tsfamily.read().await;
+                tsfamily.cache_size()
+            };
+            tsfamily
+                .write()
+                .await
+                .tsf_metrics
+                .record_cache_size(cache_size);
+        }
+    }
+
```

Comment:
deal with tsfamily drop

Reply:
Use the weak pointer upgrade to check if the Arc reference is 0 and if the tsfamily is drop.

Reply:
add test case for your code